AfricanReligions/events/oldgods_ancestor_worship_events.txt

```diff
@@ -4946,6 +5305,8 @@ character_event = {
 			OR = {
 				religion = west_african_pagan
 				religion = west_african_pagan_reformed
+				religion = amazigh
+				religion = amazigh_reformed
```

Comment:
Don't think I want Bori here? I was only using the Sun Temple event picture for the Amazigh whilst I think it would be more apt for the Bori to use the shrine event picture?

Reply:
Oh, yeah, that's fair.  But you still need to add Amazigh to the NOR.